coder-sdk/users.go

```diff
@@ -46,6 +46,14 @@ func (c Client) Me(ctx context.Context) (*User, error) {
 	return c.UserByID(ctx, Me)
 }
 
+// MyUXState updates the authenticated user's UX State.
+func (c Client) MyUXState(ctx context.Context, uxsPartial map[string]interface{}) error {
```

Comment:
Maybe this would be better as `UpdateUXState` where it accepts a `userID string` param. Then we can pass `coder.Me` where it's used.